src/lib/transport.js

```diff
@@ -27,6 +28,8 @@ function createPacketAndCondition ({
   destinationAccount,
   secret,
   data,
+  headers,
+  unsafeHeaders,
```

Comment:
The header parameters should also be included in the JSDoc of the IPR and PSK functions